server/src/parser/my_str_to_word_array.c

```diff
@@ -0,0 +1,81 @@
+/*
+** EPITECH PROJECT, 2022
+** my_teams
+** File description:
+** my_str_to_word_array
+*/
+
```

Comment:
missing /// \file comment

Reply:
needed in .c files ? not only on .h files ?

Reply:
Needed on all files